OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -178,6 +186,8 @@ private boolean shouldOverrideUrlLoadingWrapper(WebView view, String url) {
             // For more details see LEARNER-6596
             // Return false means the current WebView handles the url.
             return false;
+        } else if (isInternalLink(url)) {
```

Comment:
this check can be added at line # [183](https://github.com/edx/edx-app-android/pull/1376/files#diff-e4ab7d0704340c45605b63736de32c95R183) after that just need to update the java doc.

Reply:
IMO its a cleaner approach here to keep the conditions separate for better code readability.